util/fipstools/inject_hash_cpp/inject_hash.cpp

```diff
@@ -0,0 +1,34 @@
+#include <LIEF/LIEF.hpp>
+#include <iostream>
+#include <string>
+
+int main(int argc, char** argv) {
+    std::cout << "\n=== C++ inject_hash starting ===" << std::endl;
```

Comment:
Nothing appears to be running this, I don't see it in any of the GitHub action output.

Reply:
I think it's not running because the last few lines in `tests/ci/run_inject_hash_cpp.sh` were accidentally commented out.